stdlib/public/core/Integers.swift.gyb

```diff
@@ -2634,9 +2634,10 @@ public struct ${Self}
 
 %   if Self in ['Int32', 'Int64']:
 %     Floating = {32 : 'Float', 64 : 'Double'}[bits]
-  @_transparent
+  @available(*, unavailable,
+    message: "Please use ${Self}(bitPattern: ${Floating}) in combination with ${Floating}.bitPattern property.")
```

Comment:
This shouldn't be specifying the ${Floating}-typed argument version of the initializer since that is in fact the one you're making unavailable.